modules/reference/pages/configuration-parameters.adoc

```diff
@@ -1243,6 +1243,8 @@ backup |`nan`
 |System.Backup.S3.AWSSecretAccessKey |The secret access key for s3
 bucket |`nan`
 
+|System.Backup.S3.RoleARN |The AWS role for accessing S3 bucket, its use is prioritized over the combo of access key ID and secret access key in accessing S3. Note this is only for AWS S3, and that using role ARN assumes access key ID and secret access key are already configured with AWS CLI. |`nan`
```

Comment:
Do we need to give any examples or it should be straightforward enough for people who uses role arn?

Reply:
The usage of RoleARN is highly dependent on the node environment. Providing detailed configuration instructions might be too complex for our purposes, as it primarily pertains to AWS S3. How about we suggest users verify it first using the `aws sts assume-role` command?

Reply:
@chengjie-qin I think customers who ask about this should already be aware of this. If we were to promote what we support, we can always explain when we do the promotion. The description will make sure people who know about it knows how to make it work with TigerGraph.

Reply:
@runlilong I've updated both places with same description to make it consistent per our discussion.

Reply:
@zejunlitg @runlilong  - I definitely see value in providing a simple example with a specific persona such as DBA.

Example: Configuring S3 Backup with AWS Role ARN

Let’s walk through a scenario where Andrea, a Database Admininstrator (DBA) at a mid-sized company, is responsible for setting up automated backups of her TigerGraph cluster to an AWS S3 bucket. Instead of using long-lived credentials (AccessKeyID and SecretAccessKey), she decides to use an AWS Role ARN for enhanced security and easier management.

Step 1: Create an AWS IAM Role
Explain
Step 2: Assign the Role ARN
Explain
Step 3: Apply configuration 
Explain
Step 4: Verify the Backup configuration
Explain

Reply:
@arun-TG-PM I can add the example, but not sure where would be a suitable place to add that as these 2 places in the PR are meant to be concise?
cc @victorleeTG

Reply:
just realized there is example already in `configurations.adoc`, to help understand I'm using `arn:aws:iam::account:role/role-name-with-path` from AWS role ARN doc: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns